doc/src/content/atoms/box/props.md

```diff
@@ -7,42 +7,24 @@ title: "Box"
 
 | Prop | Description | Type | Default |
 | ------ | ----------- | ---- | ------- |
-| background | Sets the `background` CSS property | css value string | null |
-| backgroundImage | Sets the background image | css value string | null |
-| backgroundSize | Sets the background size | css value string | null |
-| bg | background color of the box | color string / from theme | null |
-| border | sets the `border` CSS property | string | null |
-| borderColor | color for the border | px string | null |
-| bottom | sets `bottom` css property | px string | null |
-| boxShadow | Sets the box shadow property | css value string | null |
-| boxSizing | box-sizing CSS property | css value string | content-box |
-| color | color of the box | color string / from theme | null |
-| display | sets the `display` CSS property | string | block |
-| height | height of the box | px string / from theme| null |
-| left | sets `left` css property | px string | null |
-| m | margin | px string / from theme| null |
-| maxHeight | maximum height of the box | px string | null |
-| maxWidth | maximum width of the box | px string | null |
-| mb | margin bottom | px string / from theme| null |
-| minHeight | minimum height of the box | px string | null |
-| minWidth | minimum width of the box | px string | null |
-| ml | margin left | px string / from theme| null |
-| mr | margin right | px string / from theme| null |
-| mt | margin top | px string / from theme| null |
-| mx | margin for left and right | px string / from theme| null |
-| my | margin for top and bottom | px string / from theme| null |
-| o | Sets the `overflow` CSS property | css value string | null |
-| ox | Sets the `overflow-x` CSS property | css value string | null |
-| oy | Sets the `overflow-y` CSS property | css value string | null |
-| p | padding for all 4 side | px string / from theme| null |
-| pb | padding bottom | px string / from theme| null |
-| pl | padding left | px string / from theme| null |
-| position | sets the `position` CSS property | string | inherit |
-| pr | padding right | px string / from theme| null |
-| pt | padding top | px string / from theme| null |
-| px | padding for left and right | px string / from theme| null |
-| py | padding for top and bottom | px string / from theme| null |
-| right | sets `right` css property | px string | null |
-| top | sets `top` css property | px string | null |
-| width | width of the box | px string / from theme| null |
-| zIndex | sets the `z-index` of the box | integer | null |
+| color | sets the color of the children texts | string or array | null |
```

Comment:
I would group the props @Kameshwaran  for example, all the margins together, etc